lib/taurus/qt/qtgui/editor/tauruseditor.py

```diff
@@ -123,27 +115,32 @@ def reload(self, idx=None, filename=None, goto=None):
     def set_current_filename(self, filename):
         self.editorStack().set_current_filename(filename)
 
+    def is_file_opened(self, filename=None):
+        # TODO: Implement it
```

Comment:
This should not be a TODO, but a base implementation (documented as such).
I mean: the current immplementation is OK. It should not be marked with a TODO. Instead, the docstring should say something like: 

Dummy implementation that always return None. Reimplement in derived classes to return the index of already-open files in the editor_stack, or None if the file is not already open.